x/axelarcork/types/codec.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+// RegisterInterfaces registers the cork proto files
+func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
+	registry.RegisterImplementations(
+		(*sdk.Msg)(nil),
+		&MsgScheduleAxelarCorkRequest{},
+		&MsgRelayAxelarCorkRequest{},
+		&MsgBumpAxelarCorkGasRequest{},
+	)
+
+	registry.RegisterImplementations((*govtypes.Content)(nil),
+		&AxelarScheduledCorkProposal{},
```

Comment:
Seems like the struct types here have the `Axelar` string appended to them, but a lot of other references to these proposals do not as mentioned earlier.

Reply:
Updated